### PR TITLE
Added the error coverage for category query

### PR DESCRIPTION
**Pull request by dineshvb**
## Purpose of this pull request

This pull request (PR) adds the error coverage for category query.

## Affected DevDocs pages

-  /guides/v2.3/graphql/queries/category.html
-  /guides/v2.4/graphql/queries/category.html

### CODE REVIEW

File: src/guides/v2.3/graphql/queries/category.md [hunk 1]
```diff
@@ -212,3 +212,10 @@ The query returns a `CategoryTree` object, which implements [`CategoryInterface`
 Attribute | Data type | Description
 --- | --- | ---
 `children` | `CategoryTree` | An array containing the next level of subcategories. By default, you can specify up to 10 levels of child categories
+
+## Errors
+
+Error | Description
+--- | ---
+`Category doesn't exist` | The specified category-id value does not exist.
```
Comment (BarnyShergold) [inline src/guides/v2.3/graphql/queries/category.md:220]:
```suggestion
`Category doesn't exist` | The specified category ID value does not exist.
```

File: src/guides/v2.3/graphql/queries/category.md [hunk 2]
```diff
@@ -212,3 +212,10 @@ The query returns a `CategoryTree` object, which implements [`CategoryInterface`
...
+`Field \"category\" argument \"id\" requires type Int, found "XXX" ` | The given category-id is not of type `int`, which is the required type.
```
Comment (BarnyShergold) [inline src/guides/v2.3/graphql/queries/category.md:221]: The suggestion is what other docs have said in this case - so consistency + should solve linting
```suggestion
`Field "category" argument "id" requires type Int, found "XXX" ` | The specified `id` argument value has the wrong type.
```

File: src/guides/v2.4/graphql/queries/category.md [hunk 3]
```diff
@@ -210,3 +210,10 @@ The query returns a `CategoryTree` object, which implements [`CategoryInterface`
 Attribute | Data type | Description
 --- | --- | ---
 `children` | `CategoryTree` | An array containing the next level of subcategories. By default, you can specify up to 10 levels of child categories
+
+## Errors
+
+Error | Description
+--- | ---
+`Category doesn't exist` | The specified category-id value does not exist.
```
Comment (BarnyShergold) [inline src/guides/v2.4/graphql/queries/category.md:218]:
```suggestion
`Category doesn't exist` | The specified category  ID value does not exist.
```

File: src/guides/v2.4/graphql/queries/category.md [hunk 4]
```diff
@@ -210,3 +210,10 @@ The query returns a `CategoryTree` object, which implements [`CategoryInterface`
...
+`Field \"category\" argument \"id\" requires type Int, found "XXX" ` | The given category-id is not of type `int`, which is the required type.
```
Comment (BarnyShergold) [inline src/guides/v2.4/graphql/queries/category.md:219]:
```suggestion
`Field "category" argument "id" requires type Int, found "XXX" ` | The specified `id` argument value has the wrong type.
```

**Review by BarnyShergold — CHANGES_REQUESTED** [review]

**Review by BarnyShergold — APPROVED** [review]
Thanks!

**Review by keharper — APPROVED** [review]

**Comment by keharper** [comment]
running tests

**Pull request merged**